backend/src/main/resources/application.properties

```diff
@@ -1,4 +1,4 @@
-quarkus.datasource.url = jdbc:postgresql://postgres:5432/postgres
+quarkus.datasource.url = jdbc:postgresql://localhost:5432/postgres
```

Comment:
don't ! this will break the deployment !t

please add `postgres 127.0.0.1` in your `/etc/hosts` file